spyder/plugins/ipythonconsole/tests/test_ipythonconsole.py

```diff
@@ -92,6 +93,19 @@ def __getattr__(self, attr):
     # Tests assume inline backend
     CONF.set('ipython_console', 'pylab/backend', 0)
 
+    # Start in a new working directory the console
+    try:
+        use_startup_wdir = request.node.get_marker('use_startup_wdir')
```

Comment:
I have question @ccordoba12 is this standard way for marker use? I was trying to look for documentation on pytest but found this instead

https://docs.pytest.org/en/latest/historical-notes.html#update-marker-code

It might not affect us or maybe it does. it seems the way to parametrize a fixture is with the use of [indirect](https://docs.pytest.org/en/latest/example/parametrize.html)

Reply:
> is this standard way for marker use?

Probably things changed between Pytest 3 (where we started to use markers) and 4+ (where there were big changes in pytest). So we should use the latest way of using markers, I'd say.

> it seems the way to parametrize a fixture is with the use of indirect

But that's to use two versions of the same fixture for every test that calls it, which I think is not the case here, right?

Reply:
The documentation in pytest is horrible, let me write an example to explain how parametrize with indirect works

Reply:
should I try to modify the fixture with pytest.mark.parametrize?

Reply:
No, no need. I just wanted to ask about this. Ignore these comments Tefa, I will open and issue to keep track of this.